class/secret-generator.yml

```diff
@@ -9,4 +15,21 @@ parameters:
           - secret-generator/component/main.jsonnet
         input_type: jsonnet
         output_path: secret-generator/
-
+      - input_type: helm
+        input_paths:
+          - secret-generator/helmcharts/kubernetes-secret-generator
+        helm_values:
+          image:
+            repository: ${secret_generator:images:secret-generator:image}
+            tag: ${secret_generator:images:secret-generator:tag}
+            pullPolicy: ${secret_generator:images:secret-generator:pullPolicy}
+        helm_params:
+          release_name: ${_instance}
+          namespace: ${secret_generator:namespace}
+        output_path: secret-generator/01_helmchart
+      # Until Kapitan supports Helm 3 charts, CRDs need to be copied explicitly
+      # https://github.com/deepmind/kapitan/pull/648
+      - input_type: copy
```

Comment:
Since this chart doesn't install any CRDs you can skip this part.

Reply:
removed